requirements.txt

```diff
@@ -5,3 +5,5 @@ six
 importlib-metadata==4.5.0
 requests
 Werkzeug~=2.0.0
+gunicorn==20.1.0
+itsdangerous==2.0.1
```

Comment:
Interesting package name!

Reply:
I don't think it's absolutely essential - all the examples I saw had it included along with gunicorn.